portal-web/docroot/html/common/themes/password_expiring_soon.jspf

```diff
@@ -14,13 +14,13 @@
  */
 --%>
 
-<c:if test="<%= themeDisplay.isSignedIn() && UserLocalServiceUtil.isPasswordExpiringSoon(user) %>">
-	<aui:script use="liferay-notice">
-		new Liferay.Notice(
+<c:if test='<%= SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

Comment:
I don't think you need to access ```themeDisplay``` in ```LoginPostAction```, I think you can just keep what you have and at line 17 above, you would just do:

```
<c:if test='<%= themeDisplay.isSignedIn() && SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

@dustinryerson/@holatuwol, do you have any thoughts on whether we should keep the ```themeDisplay.isSignedIn()```?